android/src/main/java/com/google/android/cameraview/Camera1.java

```diff
@@ -763,6 +769,21 @@ void takePictureInternal(final ReadableMap options) {
                     }
                 }
 
+                final int softwareRotation;
+                int requestedRotation = calcCameraRotation(orientationEnumToRotation(mOrientation));
+                if (requestedRotation != 0 && fallbackToSoftwareRotation()) {
+                    softwareRotation = requestedRotation;
+                    mCameraParameters.setRotation(0);
+                    try{
+                        mCamera.setParameters(mCameraParameters);
+                    }
+                    catch(RuntimeException e ) {
+                        Log.e("CAMERA_1::", "setParameters 0 rotation failed", e);
+                    }
```

Comment:
```suggestion
                    try {
                        mCamera.setParameters(mCameraParameters);
                    }
                    catch (RuntimeException e) {
                        Log.e("CAMERA_1::", "setParameters 0 rotation failed", e);
                    }
```

inconsistent use of blank space is not important for functionality but is an eyesore while reviewing :eyes:

Reply:
There are many of them. I pushed a lint version for Camera1.java

Reply:
Yeah - I noticed the whole file itself was inconsistent, I didn't note the ones where your change was at least the same as other ones, just the one that was new code. But linting in general makes me happier than it should ;-)